### PR TITLE
add docs for `config dependencies` feature

### PR DESCRIPTION
**Pull request by gsquared94**
add docs about [`multi-config`](http://tinyurl.com/skaffold-multi-configs) support

### CODE REVIEW

**Comment by gsquared94** [comment]
changes in:
- http://34.94.154.192:1313/docs/design/config/
- http://34.94.154.192:1313/docs/tutorials/config-dependencies/

File: docs/content/en/docs/design/config.md [hunk 1]
```diff
@@ -10,15 +10,74 @@ You can configure Skaffold with the Skaffold configuration file,
...
 
 You can [learn more]({{< relref "/docs/references/yaml" >}}) about the syntax of `skaffold.yaml`.
+
+## Configuration dependencies
+
+In addition to authoring pipelines in a skaffold configuration file, we can also import pipelines from other existing configurations as dependencies. Skaffold manages all imported and defined pipelines in the same session. It also ensures all artifacts in a required configs are built prior to those in current config (provided the artifacts have dependencies defined); and all deploys in required configs are applied prior to those in current config.
+
+### Local config dependency
+
+This is how a dependency added to configurations `cfg1` and `cfg2` that are defined in another `skaffold.yaml` file, looks like:
```
Comment (IsaacPD) [inline docs/content/en/docs/design/config.md:34]: Could you add that `cfg1` and `cg2` are defined in `metadata.name` of the other `skaffold.yaml`? It took me a bit to remember that's how they are defined.

**Review by IsaacPD — COMMENTED** [review]
Everything looks good, just one comment

File: docs/content/en/docs/design/config.md (hunk 1 above)
Comment (gsquared94) [reply docs/content/en/docs/design/config.md]: good point. done.

**Review by IsaacPD — APPROVED** [review]
LGTM!

**Pull request merged**